pkgs/top-level/python-packages.nix

```diff
@@ -5946,6 +5946,8 @@ in {
 
   piexif = callPackage ../development/python-modules/piexif { };
 
+  pijuice = callPackage ../development/python-modules/pijuice { };
```

Comment:
Is this to be used as an `import`able library, or there are useful command-line scripts on this package too?

Reply:
Yes, it has both.  `pijuce_cli` is a command line configuration tool.  The library also provides a \pijuice` import which allows programmatic configuration of the device.